### DISCUSSION TITLE
Composite unique field validation rule only returns error message for the last field to be evaluated

### DISCUSSION THREAD

**Discussion by lasseeee** [laravel/framework#34462] [General]
- Laravel Version: **8.5.0**
- PHP Version: 7.4.7
- Database Driver & Version: MySQL 5.7.25

### Description:
When creation a form request object with a composite unique validation rules like this:

```
            'name' => [
                Rule::unique('participants')->where(function ($query) {
                    $query->where('country_code', $this->country);
                }),
            ],
            'country_code' => [
                Rule::unique('participants')->where(function ($query) {
                    $query->where('name', $this->name);
                }),
            ],
```
and with corresponding form fields, the validation error message only shows for the last field in the validation list, `country_code` in this case. It should really display the unique validation error message for all the fields in the form, when they are attached to the fields using `$errors->first('field_name')`, not only the last one being evaluated. Is this intended behavior?

### Steps To Reproduce:
Create a form object like the one above with corresponding form fields, add per-field validation messages and submit the form.

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. 

You can post support questions on our discussions tab instead so I'm moving it over there. I won't be able to help you out personally but others who are watching this repository can help you out.

Alternatively, you can post this question on one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

Thanks.

**Comment by lasseeee** [comment]
Also the case for laravel 5 #34441, and Laravel 7.
